src/components/Modals/SlideModal/SlideModal.js

```diff
@@ -248,7 +202,9 @@ class SlideModal extends React.Component<Props, State> {
             forceInsetTop={insetTop ? 'always' : 'never'} // eslint-disable-line i18next/no-literal-string
             {...headerProps}
           />
-        }
+        ) : (
+          <HandleBar onPress={this.close} />
```

Comment:
Not sure if handle bar press should cause close, rather this should indicate the modal can be slide down to dismiss

Reply:
@IAmKio what will be the interaction?

Reply:
@divyaBacancy the defacto UX is that this handlebar should indicate that this modal can slide down to dismiss, not close outright.